### PR TITLE
Fix fund config for test env

### PR DESCRIPTION
**Pull request by Adam-W1**
fund configuration on test incorrect. The test config was incorrectly overriding the base config.

### CODE REVIEW

File: config/envs/test.py [hunk 1]
```diff
@@ -10,16 +10,4 @@ class TestConfig(DefaultConfig):
 
     # Add any test specific config here
 
-    # Database
-    SQLALCHEMY_DATABASE_URI = environ.get("DATABASE_URL").replace(
-        "postgres://", "postgresql://"
-    )
```
Comment (Adam-W1) [inline config/envs/test.py:16]: @SentientHousePlant - I wasnt sure what this was doing if we set the env var correctly. 
Do I need to keep this in?

**Comment by gidsg** [comment]
Why are there so many commits here, have you tried rebasing?

**Comment by Adam-W1** [comment]
> Why are there so many commits here, have you tried rebasing?

**Review by SentientHousePlant — APPROVED** [review]

File: config/envs/test.py (hunk 1 above)
Comment (Adam-W1) [reply config/envs/test.py:16]: okay thanks @gidsg ill keep it in

File: config/envs/development.py [hunk 2]
```diff
@@ -10,17 +10,4 @@
 class DevelopmentConfig(DefaultConfig):
 
     FSD_LOGGING_LEVEL = logging.DEBUG
-
-    SQLALCHEMY_DATABASE_URI = environ.get("DATABASE_URL")
```
Comment (gidsg) [inline config/envs/development.py:14]: I think we want to keep this in to let you override the database locally?

File: config/envs/test.py (hunk 1 above)
Comment (Adam-W1) [reply config/envs/test.py]: okay thanks both. So if PAAS does pass it in correctly, its still safe to have this code in - ill leave it in 👍

File: config/envs/development.py (hunk 2 above)
Comment (SentientHousePlant) [reply config/envs/development.py:14]: its in the default env so we shouldn't need to have it here too

Comment (Adam-W1) [reply config/envs/development.py:14]: agreed 👍

**Review by gidsg — APPROVED** [review]
LGTM 👍 be sure to squash and merge.

**Pull request merged**